class .js

```diff
@@ -0,0 +1,38 @@
+
+////////////////////////////Product 클래스 구현
+
+class product{
```

Comment:
style: 클래스명은 일반적으로 PascalCase를 따릅니다.
```suggestion
class Product{
```